platforms/hyperledger-fabric/charts/fabric-chaincode-invoke/templates/invoke_chaincode.yaml

```diff
@@ -192,6 +192,10 @@ spec:
         args:
         - |-
           #!/bin/bash sh
+          if [ "${INIT_REQUIRED}" = "false" ]
+          then
+             INVOKE_ARGUMENTS='\"init\",\"\"'
```

Comment:
`init` function is very subjective to the chaincode, and it may vary from chaincode to chaincode. Ideally in the case of `INIT_REQUIRED` as false chaincode invocation can be skipped.

Reply:
Done